jobs/config.json

```diff
@@ -2539,27 +2539,27 @@
       "UNKNOWN"
     ]
   },
-  "ci-kubernetes-e2e-gce-master-new-cvm-kubectl-skew": {
+  "ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew": {
     "args": [
       "--check-leaked-resources",
       "--check-version-skew=false",
       "--env-file=jobs/platform/gce.env",
-      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-cvm-kubectl-skew.env",
+      "--env-file=jobs/ci-kubernetes-e2e-gce-master-new-gci-kubectl-skew.env",
       "--extract=ci/latest-1.7",
       "--extract=ci/latest",
-      "--gcp-project=gce-cvm-upg-1-5-1-4-ctl-skew",
+      "--gcp-project=gce-gci-upg-1-5-1-4-ctl-skew",
```

Comment:
I think it should work as is not.  Mind if I do that as a follow up to reduce the number of things I am changing at once?